ddtrace/settings/config.py

```diff
@@ -577,7 +578,7 @@ def __getattr__(self, name) -> Any:
     def _add_extra_service(self, service_name: str) -> None:
         if self._extra_services_queue is None:
             return
-        if self._remote_config_enabled and service_name != self.service:
+        if self._remote_config_enabled and service_name != self.service and not gevent_is_patched():
```

Comment:
Should not be needed as the queue should be None there anyway.

Reply:
we have to check at runtime because gevent patching will probably happen _after_ importing ddtrace and the creation of this config instance :/